src/update-template.js

```diff
@@ -19,7 +19,6 @@ const { getFromRegistry, updateInRegistry } = require('./registry');
             "name": npmPackageMetadata.name,
             "description": npmPackageMetadata.description,
             "latestVersion": npmPackageMetadata.version,
-            "services": npmPackageMetadata.services,
```

Comment:
The same as for `src/add-to-registry.js` we need to change "services" to "categories" as categories are mandatory in `install.yml`. And update [this block](https://github.com/adobe/aio-template-submission/blob/c7d7b9d955ee46f870361d9278ab0eef3b5b8646/src/update-template.js#L33-L35) to
```
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```

We will have
```
        let templateData = {
            "author": npmPackageMetadata.author,
            "name": npmPackageMetadata.name,
            "description": npmPackageMetadata.description,
            "latestVersion": npmPackageMetadata.version,
            "categories": npmPackageMetadata.categories,
            "adobeRecommended": adobeRecommended,
            "keywords": npmPackageMetadata.keywords,
            "links": {
                "npm": npmUrl,
                "github": gitHubUrl
            }
        };
        if (npmPackageMetadata.extension) {
            templateData['extension'] = npmPackageMetadata.extension;
        }
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```